routes.py

```diff
@@ -232,3 +236,147 @@ async def delete_time_slot(id: str):
+            )
+            # print(meeting_tile)
+            meetings.append(meeting_tile)
+        
+    return schemas.MeetingTileCollection(meetings=meetings)
+
+@app.get(
+    "/meetings/{id}",
+    response_description="Get a single meeting",
+    response_model=models.MeetingModel,
+    response_model_by_alias=False,
+)
+async def show_meeting(id: str):
+    if (meeting := await meetings_collection.find_one({"_id": ObjectId(id)})) is not None:
+        return meeting
+    
+    raise HTTPException(status_code=404, detail=f"meeting {id} not found")
+
+@app.patch(
+    "/invites/{id}",
+    response_description="Change status of invitation",
+    response_model=models.MeetingInvite,
+    response_model_by_alias=False,
+)
+async def change_invite_status(id: str, status: schemas.UpdateMeetingStatus = Body(...), user: schemas.AuthSchema = Depends(JWTBearer())):
+    print(status)
+    print(models.MeetingStatus.__members__)
+    if (status.status not in models.MeetingStatus.__members__):
+        raise HTTPException(status_code=400, detail="Invalid status")
+    user_found = await user_collection.find_one({"_id": ObjectId(user.id)})
+    if user_found is None:
+        raise HTTPException(status_code=404, detail="Account not found")
+    meeting = await meetings_collection.find_one({"_id": ObjectId(id)})
+    if meeting is None:
+        raise HTTPException(status_code=404, detail=f"Meeting {id} not found")
+    
+    meeting_invites = user_found.get("meetings", [])
+    for invite in meeting_invites:
+        if invite["meeting_id"] == id:
+            invite["status"] = status.status
+            await user_collection.update_one(
+                {"_id": user_found["_id"]},
+                {"$set": {"meetings": meeting_invites}}
+            )
+            return invite
+
+@app.patch(
+    "/meetings/{id}",
+    response_description="Update a meeting",
+    response_model=models.MeetingModel,
+    response_model_by_alias=False,
+)
+async def update_meeting(id: str, meeting: schemas.UpdateMeeting = Body(...)):
+    meeting_dict = {
+        k: v for k, v in meeting.model_dump(by_alias=True).items() if v is not None
+    }
+
+    if len(meeting_dict) >= 1:
+        update_result = await meetings_collection.find_one_and_update(
+            {"_id": ObjectId(id)},
+            {"$set": meeting_dict},
+            return_document=ReturnDocument.AFTER,
+        )
+        if update_result is not None:
+            return update_result
+        else:
+            raise HTTPException(status_code=404, detail=f"meeting {id} not found")
+
+    if (existing_meeting := await meetings_collection.find_one({"_id": id})) is not None:
+        return existing_meeting
+
+    raise HTTPException(status_code=404, detail=f"meeting {id} not found")
+
+@app.delete(
+    "/meetings/{id}", 
+    response_description="Delete a meeting"
+)
+async def delete_meeting(id: str):
+    delete_result = await meetings_collection.delete_one({"_id": ObjectId(id)})
```

Comment:
We should also probably delete all invites created for the meeting, maybe by storing invited users in the meeting model